molecule/default/molecule.yml

```diff
@@ -23,6 +23,9 @@ platforms:
       - 0.0.0.0:8080:8080/TCP
 provisioner:
   name: ansible
+  env:
+    PROD_JBOSSNETWORK_API_CLIENTID: "${PROD_JBOSSNETWORK_API_CLIENTID}"
+    PROD_JBOSSNETWORK_API_SECRET: "${PROD_JBOSSNETWORK_API_SECRET}"
```

Comment:
@RanabirChakraborty I don't think you need that. This part of the Molecule file is to **add** env vars that are not already defined. While it could feel clear to add these here, I think it's better to hide this complexity from the demo code. (especially as user will probably provides those creds using the -e @file.yml option anyway).